plugins/helloworld/fps_helloworld/exceptions.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import Request, Response
+from fastapi.responses import RedirectResponse
+
+from fps.hooks import register_exception_handler
+from fps.logging import get_configured_logger
+
+logger = get_configured_logger("helloworld")
+
+
+class RedirectException(Exception):
```

Comment:
while this is more flexible, I thought adding a generic `RedirectException` to fps itself could have been useful. In that scenario no new hook would have been necessary (IIUC).

Then consumers could have just `from fps.exceptions import RedirectException` and then do whatever. 

Now everyone has to come up with their own RedirectException.

Reply:
I guess! You're the expert